src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -88,6 +90,20 @@ object TryUtil {
     }
   }
 
+  def defaultRetry[T](f: => T, logger: WorkflowLogger, failureMessage: String, fatalExceptions: Seq[Class[_ <: Throwable]] = Seq.empty) = {
```

Comment:
Do we really want to be adding to the mess that is the tryutil retry functionality?